Core/System.cpp

```diff
@@ -97,13 +97,19 @@ bool coreCollectDebugStatsForced = false;
 
 // This can be read and written from ANYWHERE.
 volatile CoreState coreState = CORE_STEPPING;
-// Note: intentionally not used for CORE_NEXTFRAME.
+// Note: intentionally not used for CORE_NEXTFRAME. << TODO: This comment is a lie, it is used!
```

Comment:
Hm, I no longer remember why this was intentional anyway...
https://github.com/hrydgard/ppsspp/commit/2164a7fdf9e5ea476e693554b4f8c94769230cac#diff-16696ea83f1842f3ac2352345fc716beR41

-[Unknown]